build.sbt

```diff
@@ -18,6 +18,7 @@ libraryDependencies += "com.databricks" % "dbutils-api_2.12" % "0.0.5" % Provide
 libraryDependencies += "com.amazonaws" % "aws-java-sdk-s3" % "1.11.595" % Provided
 libraryDependencies += "io.delta" % "delta-core_2.12" % "1.0.0" % Provided
 libraryDependencies += "org.scalaj" %% "scalaj-http" % "2.4.2"
+libraryDependencies += "com.lihaoyi" %% "sourcecode" % "0.4.1"
```

Comment:
@souravbaner-da, I ust noticed your comment from 2024-06-05.  I discussed this with @sriram251-code [here](https://databricks.slack.com/archives/C051AFDP6LT/p1717508580671179) in Slack back on 2024-06-04.  I believe we have discussed it on >= 1 team calls since then also.  If it's still a concern I would like to learn more about the standard you mention.  Is that in a document somewhere?